src/ElectroGrammar.g4

```diff
@@ -5,6 +5,10 @@
 grammar ElectroGrammar;
 import Passive, Semi;
 
-electro_grammar: (passive | semi) EOF;
+electro_grammar: (passive | semi) SPECIAL_CHARS? EOF;
 
 WHITESPACE: [\p{White_Space}] -> skip;
+
+THROUGHHOLE: T H R O U G H '-'? H O L E -> skip;
```

Comment:
Seems like you want a more lax parser ;)

Reply:
Yeah just started hacking on the lax parser, commit is already dropped ;)